scripts/approve-listings-by-user.js

```diff
@@ -0,0 +1,66 @@
+// This dotenv import is required for the `.env` file to be read
+require('dotenv').config();
+
+const flexIntegrationSdk = require('sharetribe-flex-integration-sdk');
+
+// Read user email from arguments
+const email = process.argv[2];
+
+if (!email) {
+  console.log('User email is missing, one needs to be provided as the first argument, exiting.');
+  process.exit(1);
+}
+
+const integrationSdk = flexIntegrationSdk.createInstance({
+
+  // These two env vars need to be set in the `.env` file.
+  clientId: process.env.FLEX_INTEGRATION_CLIENT_ID,
+  clientSecret: process.env.FLEX_INTEGRATION_CLIENT_SECRET,
+
+  // Normally you can just skip setting the base URL and just use the
+  // default that the `createInstance` uses. We explicitly set it here
+  // for local testing and development.
+  baseUrl: process.env.FLEX_INTEGRATION_BASE_URL || 'https://flex-api.sharetribe.com',
+});
+
+// Query params for listings query
+const listingsQueryParams = authorId => ({
+  authorId,
+  states: ['pendingApproval'],
+  'fields.listing': 'state',
+});
+
+// Query params for approve listing request
+const listingsApproveParams = {
+  expand: true,
+  'fields.listing': 'title',
+};
+
+// query params for users/show request
+const usersShowParams = emailAddress => ({
+  email: emailAddress,
+  'fields.listing': 'none',
```

Comment:
Is it necessary to pass the `'fields.listing': 'none'` as param? In user metadata example we don't do that so what's the difference?

Reply:
It is supposed to narrow down the returned attributes, but now I see that it's targeting `listing` fields and our `show` endpoints don't even support the sparse attributes so I'll remove it. Probably just left there from some testing.

Reply:
Also, this was not even used 🤦‍♂.